plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/FiberChannelAdapter.java

```diff
@@ -16,13 +16,39 @@
 // under the License.
 package com.cloud.hypervisor.kvm.storage;
 
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import org.apache.log4j.Logger;
+
 import com.cloud.storage.Storage;
 import com.cloud.utils.exception.CloudRuntimeException;
 
 @StorageAdaptorInfo(storagePoolType=Storage.StoragePoolType.FiberChannel)
 public class FiberChannelAdapter extends MultipathSCSIAdapterBase {
+
+    private Logger LOGGER = Logger.getLogger(getClass());
+
+    private String hostname = null;
+    private String hostnameFq = null;
+
     public FiberChannelAdapter() {
         LOGGER.info("Loaded FiberChannelAdapter for StorageLayer");
+        // get the hostname - we need this to compare to connid values
+        try {
+            InetAddress inetAddress = InetAddress.getLocalHost();
+            hostname = inetAddress.getHostName(); // basic hostname
+            if (hostname.indexOf(".") > 0) {
+                hostname = hostname.substring(0, hostname.indexOf(".")); // strip off domain
+            }
+            if (hostname.indexOf(".") > 0) {
+                hostname = hostname.substring(0, hostname.indexOf(".")); // strip off domain
+            }
```

Comment:
substring twice here, is this intentional?

Reply:
addressed in commit 42780e76163f72e7663b78945e74e62325fb4158 and 018e55678d494bdb53f1385fba57bd84031ae4a4